mailpoet/assets/js/src/form_editor/utils/link_suggestions.tsx

```diff
@@ -2,7 +2,7 @@ import apiFetch from '@wordpress/api-fetch';
 import { addQueryArgs } from '@wordpress/url';
 import { decodeEntities } from '@wordpress/html-entities';
 import MailPoet from 'mailpoet';
-import type {WP_REST_API_Search_Results} from 'wp-types';
+import type { WP_REST_API_Search_Results } from 'wp-types';
```

Comment:
Ah, I see this got somehow fixed in an unrelated commit. Not all spacing seems to be fixed though.